source/getting-started.html.md

```diff
@@ -0,0 +1,256 @@
+
+If you take your laptop home at night, you may want to download the data while in the office and restore it overnight to minimise disruption.
+
+First, do the download on your host as the unzipping is a lot quicker when not run over NFS:
+
+    mac$ ./replicate-data-local.sh -u your_ssh_username -n
+
+Then when you get home (or if you have a spare hour during meetings) run the script on your VM and specify the backup directory for the date you performed the download:
+
+    dev$ ./replicate-data-local.sh -s -d backups/2016-11-17
+
+## 7. Run your apps
+
+> You can run any of the GOV.UK apps from the `/var/govuk/govuk-puppet/development-vm` directory. You’ll need to run `bundle install` in this folder.
+
+### Run a single app
+
+You can use [foreman](http://ddollar.github.io/foreman/) to run a single app. The available apps are defined in the Procfile.
+
+    dev$ cd /var/govuk/govuk-puppet/development-vm
+    dev$ foreman start rummager
+
+### Run multiple apps
+
+Since many of our apps depend on other apps, we normally run them using [bowler](https://github.com/JordanHatch/bowler) instead of foreman.
+
+To run particular apps with bowler, use:
+
+    dev$ bowl content-tagger
+
+This will also run all of the dependencies defined in the Pinfile.
+
+If you don't need an optional dependency, you can pass the `-w` option:
+
+    dev$ bowl whitehall -w mapit
+
+If you want to run the project in development mode with the static assets served from your local copy, run bowler with the STATIC_DEV variable defined and make sure you're not setting static=0:
+
+    dev$ STATIC_DEV="http://static.dev.gov.uk" bowl planner static
+
+## 8. Access remote environments
+
+### Access the web frontend
+
+Most GOV.UK web applications and services are available via the public internet, on the following forms of URL:
+
+* [https://www-origin.integration.publishing.service.gov.uk](https://www-origin.integration.publishing.service.gov.uk) (integration, HTTP basic auth)
+* [https://deploy.staging.publishing.service.gov.uk](https://deploy.staging.publishing.service.gov.uk) (staging, restricted to GDS office IP addresses)
+* [https://alert.publishing.service.gov.uk](https://alert.publishing.service.gov.uk) (production, restricted to GDS office IP addresses)
+
+The basic authentication username and password is widely known, so just ask somebody on your team if you don't know it.
+
+### Access servers via SSH
+
+To access servers via SSH:
+
+1. Add your machine’s public key to the Puppet repository.
+2. Deploy the Puppet repository to the environment you're trying to access.
+
+While the applications are available directly via the public
+internet, SSH access to the environment is via a ‘jumpbox’. You’ll need to configure your machine to use this jumpbox. See
+the [instructions in the Dev Manual](https://docs.publishing.service.gov.uk/manual/technical-setup.html#ssh-config) for adding the relevant lines to your `~/.ssh/config`.
```

Comment:
Maybe we can import the SSH part of that page into here? Then we can remove that page probably.